website/docs/r/virtual_machine.html.markdown

```diff
@@ -343,7 +130,7 @@ The following arguments are supported:
 * `vm_size` - (Required) Specifies the [size of the virtual machine](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-size-specs/).
 * `storage_image_reference` - (Optional) A Storage Image Reference block as documented below.
 * `storage_os_disk` - (Required) A Storage OS Disk block as referenced below.
-* `delete_os_disk_on_termination` - (Optional) Flag to enable deletion of the OS disk VHD blob or managed disk when the VM is deleted, defaults to `false`
+* `delete_os_disk_on_termination` - (Optional) Flag to enable deletion of the OS disk VHD blob or managed disk when the VM is deleted, defaults to `false` (not yet supported).
```

Comment:
I think this'd be better as `Flag to enable deletion of the OS disk VHD blob when the VM is delete`?d

Reply:
You are right, let me change the wording.